js/modules/types/attachment/migrate_data_to_file_system.js

```diff
@@ -7,15 +7,15 @@ const {
 
 
 // type Context :: {
-//   writeAttachmentData :: ArrayBuffer -> Promise (IO Path)
+//   writeNewAttachmentData :: ArrayBuffer -> Promise (IO Path)
```

Comment:
Added `*New*` infix to distinguish saving a new attachment (create random file name / path) vs saving an existing attachment (reuse existing `attachment.path`).